jme3-core/src/main/resources/Common/MatDefs/Light/GBuf.vert

```diff
@@ -70,4 +70,4 @@ void main(){
     #ifdef VERTEX_COLOR
       DiffuseSum *= inColor;
     #endif
-}
\ No newline at end of file
+}

```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR